packages/babel-helpers/scripts/generate-helpers.js

```diff
@@ -3,8 +3,7 @@ import fs from "fs";
 import { join } from "path";
 import { URL, fileURLToPath } from "url";
 import { minify } from "terser";
-import { transformSync } from "@babel/core";
-import presetTypescript from "@babel/preset-typescript";
+import { babel, presetTypescript } from "$repo-utils/babel-top-level";
```

Comment:
This is because now this package has a `devDependency` on `@babel/core@workspace` (for types), but we need the root dependency to compile.